README.md

```diff
@@ -5,7 +5,8 @@ Practice space for using the Github Workflow.
 
 Participants:
 Ben Coleman
-
+Dylan Fodor
```

Comment:
It would be nice to have everyone'e email address with the names.

Reply:
Agreed! It would be nice